teuthology/packaging.py

```diff
@@ -26,6 +27,17 @@
     'httpd': {'deb': 'apache2', 'rpm': 'httpd'}
 }
 
+DISTRO_CODENAME_MAP = {
+    "ubuntu": {
+        "14": "trusty",
+        "12": "precise",
+        "15": "vivid",
```

Comment:
well.....there are two releases per year; 12.04 is precise, 12.10 is quantal, 13.04 is raring, 13.10 saucy, etc.  Within our limited sphere the major-only compare for the only minor we care about works, but it's confusing; I'd use the full version, even if you leave holes.